core/src/test/scala/unit/kafka/server/ShareFetchAcknowledgeRequestTest.scala

```diff
@@ -54,7 +55,7 @@ class ShareFetchAcknowledgeRequestTest(cluster: ClusterInstance) extends GroupCo
     )
 
     val shareFetchRequest = createShareFetchRequest(groupId, metadata, send, Seq.empty, Map.empty)
-    val shareFetchResponse = connectAndReceive[ShareFetchResponse](shareFetchRequest)
+    val shareFetchResponse = connectAndReceiveWithoutClosingSocket[ShareFetchResponse](shareFetchRequest)
```

Comment:
So the reason for all to be on open socket is that there session will be release, but it should not be a problem always, correct? So do you need to change every method here?

Reply:
Yeah, it's fine.